minimalTest.js

```diff
@@ -92,18 +104,24 @@ async function start() {
         setRoomList();
     })
 
-    client.on("Room.timeline", (event, room, toStartOfTimeline) => {
+    client.on("Room.timeline", (event, room, toStartOfTimeline, _removed, data) => {
```

Comment:
what do these params do?

Reply:
i thought [data.liveEvent](https://matrix-org.github.io/matrix-js-sdk/23.3.0/interfaces/IRoomTimelineData.html#liveEvent) could be used to determine whether an event was live or not instead of using the timestamp trick, but it seems to be true for every single event

Reply:
reverted